scripts/spi_flash/board_defs.py

```diff
@@ -65,6 +65,14 @@
         'cs_pin': "PC11",
         'skip_verify': True
     },
+    'btt-octopus-pro-h723-v1': {
```

Comment:
@locki-cz  -- after you rebase your PR to the current master branch, you should be able to drop this extra entry and just add an alias for the octopus pro as shown below.